apps/security-advisory-patches-portal/webapp/src/index.css

```
@@ -0,0 +1,50 @@
+herein in any form is strictly forbidden, unless permitted by WSO2 expressly.
+You may not alter or remove any copyright or other notice from copies of this content.
+*/
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: 'Roboto', -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Oxygen',
```

Stylelint (17.11.0)

[error] Expected no quotes around "Roboto" (font-family-name-quotes)

(font-family-name-quotes)